src/mysql/MySQLClient.ts

```diff
@@ -1,13 +1,17 @@
-import * as mysql from 'mysql';
-import { createPool, Pool, Factory, Options } from 'generic-pool';
-import { DBTransaction } from '../db/DBTransaction';
-import { ConnectionPool, PoolConfig, ConnectionConfig } from '../db/ConnectionPool';
-import { Transaction } from '../transaction/TransactionManager';
-import { LogManager } from '../log/LogManager';
-import { DBClient, DBClientConfig } from '../db/DBClient';
+import * as mysql from "mysql";
+import { createPool, Pool, Factory, Options } from "generic-pool";
+import { DBTransaction } from "../db/DBTransaction";
+import {
+  ConnectionPool,
+  PoolConfig,
+  ConnectionConfig
+} from "../db/ConnectionPool";
+import { Transaction } from "../transaction/TransactionManager";
+import { LogManager } from "../log/LogManager";
+import { DBClient, DBClientConfig } from "../db/DBClient";
```

Comment:
If double quotes is changed by prettier.io, prettier is better to be added to package.json as DOC suggested.